hooks/pre_commit/09-version_locking.rb

```diff
@@ -8,6 +8,10 @@
   kafo.config.configure_application
 end
 
+if lock_versions && !foreman_maintain_installed?
+  fail_and_exit('Locking of package versions was requested but foreman-maintain is not installed')
+end
```

Comment:
I think this is not needed because `package_lock_feature?` implies `foreman_maintain_installed?`